app/src/main/res/values-night/colors_dark.xml

```diff
@@ -27,6 +27,7 @@
     <item name="colorChatInputButton" type="color">@color/button_background_dark</item>
     <item name="colorChatHedvigBackground" type="color">@color/button_background_dark</item>
 
+    <item name="colorLargeButton" type="color">@color/violet_300</item>
```

Comment:
Should this color be only for large buttons, or all buttons?

Reply:
from the design it looks like only large buttons

Reply:
Consistency police 🚓